Dockerfile

```diff
@@ -1,11 +1,10 @@
-FROM openjdk:17-jdk-slim
+FROM eclipse-temurin:17-jdk-alpine
 
 ARG JAR_FILE=ssdc-rm-caseprocessor*.jar
-
-CMD ["/usr/local/openjdk-17/bin/java", "-jar", "/opt/ssdc-rm-caseprocessor.jar"]
+CMD ["/opt/java/openjdk/bin/java", "-jar", "/opt/ssdc-rm-caseprocessor.jar"]
 COPY healthcheck.sh /opt/healthcheck.sh
-RUN groupadd --gid 999 caseprocessor && \
-    useradd --create-home --system --uid 999 --gid caseprocessor caseprocessor
+RUN addgroup --gid 99 caseprocessor && \
+    adduser --system --uid 99 caseprocessor caseprocessor
```

Comment:
I believe the UID and GID here should be => 1000 and lower values can be reserved and might create issues mapping volumes etc. I'm not 100% sure of the details on Alpine but we've had issues with this is in the past and avoiding the lower, reserved IDs helped.

Reply:
@KieranWardle @AdamHawtin i seem to remember reading that for GKE we should be using a UID > 2000 so we don't clash with the GKE host users, will see if i can find the docs